code/API_definitions/qod-api.yaml

```diff
@@ -433,6 +493,23 @@ components:
             - expiresAt
             - qosStatus
 
+    SessionInfoList:
+      description: Paged response of active sessions created by the user
```

Comment:
Here we mention created by the user, and above created by the client. This topic is very relevant for this endpoint as discussed in #101. The logic must be to return the sessions that are allowed to be discovered by the access token. In the most restrictive case, for three-legged tokens, that includes checking both client_id and user. 
Returning just sessionIds or all session details does not change the picture, if knowing sessionIds is enough to get session details in a separate operation.